libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -30,7 +30,14 @@ export class BookSearchComponent {
     private readonly store: Store,
     private readonly fb: FormBuilder
   ) {}
-
+  ngOnInit(): void {
+    this.searchForm
+      .get('term')
+      .valueChanges.pipe(debounceTime(OkReadsConstants.time), takeUntil(this.destroyed$))
```

Comment:
I would store the subscription object and manually create the subscription so it is less prone to errors if `takeUntil` is in the wrong place.